components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -105,3 +113,28 @@ private fun determineVerticalOffset(orientation: BrowserMenu.Orientation, view:
         -view.measuredHeight
     }
 }
+
+/**
+ * Indicates in which orientation, a menu should be layout.
+ */
+internal val View.anchoredMenuOrientation: BrowserMenu.Orientation
+    get() {
+        return if (isCloseToTheBottomOfTheScreen) {
+            BrowserMenu.Orientation.UP
+        } else {
+            BrowserMenu.Orientation.DOWN
+        }
+    }
+
+private val View.isCloseToTheBottomOfTheScreen: Boolean
+    get() {
+        val screenHeight = context.resources.displayMetrics.heightPixels
+        return screenLocationY > (screenHeight / 2)
```

Comment:
Does using `heightPixels` still work when using split screen mode? Since the other value is "location in window" I wonder if you need the height of the "window" here.

Reply:
I tested using `heightPixels` with the split screen mode and it returns the actual height of the windows. For example, I changed the app size and it was getting updated as I was increasing or reducing the app size.